src/components/VirtualizedTreeSelect.js

```diff
@@ -180,11 +241,12 @@ VirtualizedTreeSelect.defaultProps = {
   optionHeight: 25,
   optionLeftOffset: 16,
   expanded: false,
-  isMenuOpen: false,
+  isMenuOpen: undefined,
```

Comment:
Same as in `IntelligentTreeSelect` - why change it from `false` to `undefined`. The code does not seem to require it.